sklearn/tree/_export.py

```diff
@@ -212,6 +212,7 @@ def __init__(self, max_depth=None, feature_names=None,
         self.rounded = rounded
         self.precision = precision
         self.fontsize = fontsize
+        self.fontname = fontname
```

Comment:
I think `self.fontname` should be set in `_DOTTreeExporter.__init__`, because `_BaseTreeExporter` does not use `self.fontname`.